enterprise/app/models/sla_policy.rb

```diff
@@ -3,10 +3,12 @@
 # Table name: sla_policies
 #
 #  id                         :bigint           not null, primary key
-#  frt_threshold              :float
+#  first_response_time        :float
```

Comment:
I believe the naming would okay because SLA models are supposed to save only the threshold. Nothing else. However, would it be easier to have threshold in the name for a quick search. 

The `first_response_time` search would return 507 results from 123 files.
<img width="339" alt="Screenshot 2024-01-19 at 10 05 43 AM" src="https://github.com/chatwoot/chatwoot/assets/2246121/f35f5fe2-b60c-42e2-b2f6-d6e754ff28d5">

Reply:
True. I will make changes accordingly.